packages/core/react-loosely-lazy/src/profiler/index.ts

```diff
@@ -0,0 +1,25 @@
+import { createContext } from 'react';
+
+export type EventInfo = {
+  identifier: string;
+};
+
+export type Profiler = {
+  onLoadStart?(info: EventInfo): void;
+  onLoadComplete?(info: EventInfo): void;
+};
+
+let globalInstance: Profiler | undefined;
+export function setGlobalReactLooselyLazyProfilerInstance(instance: Profiler) {
+  globalInstance = instance;
+}
+
+const globalInstanceProxy = {
+  onLoadStart: (info: EventInfo) => globalInstance?.onLoadStart?.(info),
+  onLoadComplete: (info: EventInfo) => globalInstance?.onLoadComplete?.(info),
+};
+
+export const GlobalReactLooselyLazyProfiler: Readonly<Profiler> =
+  globalInstanceProxy;
```

Comment:
The common way to do this trick is to put it in a `Box<T>` / `Ref<T>` style container. E.g.
```js
export const GlobalReactLooselyLazyProfiler = {current: null}
```
Then it's mutable
```js
GlobalInstance.current = { onLoadStart: () => {}, onLoadComplete: () => {}}
```
And accesses look like this
```js
const Profiler = GlobalInstance.current
if (Profiler != null){
  Profiler.onLoadStart()
}
```